Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.m

```diff
@@ -112,13 +112,6 @@ - (void)handleRequest:(BTPaymentFlowRequest *)request
 
         NSError *integrationError;
 
-        if (!NSClassFromString(@"CardinalSession")) {
-            [[BTLogger sharedLogger] critical:@"CardinalMobile framework is not included. For SPM integrations, you must manually include it."];
-            integrationError = [NSError errorWithDomain:BTThreeDSecureFlowErrorDomain
-                                                   code:BTThreeDSecureFlowErrorTypeConfiguration
-                                               userInfo:@{NSLocalizedDescriptionKey: @"CardinalMobile framework is not included. For SPM integrations, you must manually include it."}];
-        }
```

Comment:
Merchants will get a compile-time error if Cardinal hasn't been included correctly, so this run-time error will never be sent. It should be safe to remove it.